src/Sarif.Multitool/Sarif.Multitool.csproj

```diff
@@ -14,20 +14,7 @@
     <StartupObject></StartupObject>
   </PropertyGroup>
 
-  <!--NuGet Config-->
-  <PropertyGroup>
-    <IsTool>true</IsTool>
-    <PackageId>Sarif.Multitool</PackageId>
-    <Title>Microsoft SARIF Multitool (includes SARIF SDK)</Title>
-    <Authors>Microsoft Corporation</Authors>
-    <Description>SARIF-based classes and utilities for building static analysis drivers. Includes the SARIF SDK.</Description>
-    <PackageReleaseNotes>Version $(Version) of the .NET SARIF SDK (for SARIF v1.0.0)</PackageReleaseNotes>
-    <Copyright>Copyright Microsoft 2015</Copyright>
-    <PackageLicenseUrl>https://github.com/Microsoft/sarif-sdk/blob/master/LICENSE</PackageLicenseUrl>
-    <PackageProjectUrl>https://github.com/microsoft/sarif-sdk</PackageProjectUrl>
-    <PackageTags>command line driver utilities static analysis sarif framework utils driver.utilities</PackageTags>
-    <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
-  </PropertyGroup>
+  <!--NuGet metadata located in src\Nuget\Sarif.Multitool.nuspec-->
```

Comment:
I don't think this comment is necessary. I don't think anybody reading the project file will wonder "Where did the NuGet metadata go?" because they won't know it was here in the first place. And they won't be surprised to see a .nuspec file on the file system, and they'll know to look there for the metadata.

Reply:
Removed